app/src/main/java/com/example/benchproject/data/movie/details/repo/RepositoryModule.kt

```diff
@@ -0,0 +1,15 @@
+package com.example.benchproject.data.movie.details.repo
+
+import com.example.benchproject.domain.movie.details.repo.MovieDetailsRepository
+import dagger.Binds
+import dagger.Module
+import dagger.hilt.InstallIn
+import dagger.hilt.android.components.ViewModelComponent
+
+@Module
+@InstallIn(ViewModelComponent::class)
+interface RepositoryModule {
```

Comment:
You already have one `RepositoryModule`, why you are creating another one with the same (generic) name?

Reply:
Ok, maybe I should change both file names or make the same module for these both repositories in movies.common?

Reply:
I did it :D